### PR TITLE
Search Source Setting guide text

### PR DESCRIPTION
**Pull request by onesounds**
from https://github.com/Flow-Launcher/Flow.Launcher/issues/1309

**Before (and other languages)** 

**After**

Currently, the text is separated into three paragraphs, and inline bold cannot be used. To produce as requested, the design must be changed to four paragraphs, and all existing translations must be changed.

Therefore, it becomes the current editing form, and it is necessary to check if it is okay. Other languages will remain in their existing form, leaving each language worker at their discretion to translate according to the layout.

### CODE REVIEW

File: Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml [hunk 1]
```diff
@@ -75,10 +75,11 @@
                                 Text="{DynamicResource flowlauncher_plugin_websearch_guide_1}"
                                 TextAlignment="Left"
                                 TextWrapping="WrapWithOverflow" />
-                            <TextBlock
+                            <TextBox
```
Comment (taooceros) [inline Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml:78]: what's this for?

Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml:78]: for selectable text. if user want to test, user can simply drag the url.

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml:78]: I didn't get the idea? Isn't Textblock also selectable?

Comment (onesounds) [reply Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml:78]: normal textblock can't selectable. (maybe there are way but I didn't checked.) but textbox more look like selectable. if you think looks weired. I'll change it.  😘

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.WebSearch/SearchSourceSetting.xaml:78]: Na it's OK. i am just curious about what it is for.

**Review by taooceros — APPROVED** [review]

**Comment by dracp** [comment]
Hello.
I finish translating into Spanish, the three new strings listed below.
However, excuse the insistence.
I think the text is redundant. The same action of copying the content of the address bar is indicated twice, and the same action of substituting the test string for {q} twice. Once in the first paragraph and again in the third paragraph.

_If you want to add a search for a particular website to Flow, first enter a dummy text string in the search bar of that website, and launch the search. **Now copy the contents of the browser's address bar, and paste it in the URL field below. Replace your test string with {q}**. For example, if you search for casino on Netflix, its address bar reads

-https://www.netflix.com/search?q=Casino

-**Now copy this entrire string and paste it in the URL field below.**
        **Then replace casino with {q}.**
        Thus, the generic formula for a search on Netflix is https://www.netflix.com/search?q={q}

**### Alternative text proposal:**

If you want to add a search for a particular website to Flow, first enter a dummy text string in the search bar of that website, and launch the search, for example search for "Casino" in "Netflix".

Now copy the contents of the browser's address bar, https://www.netflix.com/search?q=Casino, and paste it in the URL field below, replacing your test string, Casino, with {q}. Thus, the generic formula for a search on Netflix is https://www.netflix.com/search?q={q}

**Comment by jjw24** [comment]
Hi @dracp, I see the redundancy in the two areas of the paragraph. I have added a new issue with your comment.

**Comment by dracp** [comment]
> Hi @dracp, I see the redundancy in the two areas of the paragraph. I have added a new issue with your comment.

In FL 10.0.0. November 28, the text in the Search Source Setting box is still redundant. 
I understand that some of the text in the first paragraph should be deleted:

"If you want to add a search for a particular website to Flow, first enter a dummy text string in the search bar of that website, and launch the search. Now copy the contents of the browser's address bar, and paste it in the URL field below. Replace your test string with {q}. For example, if you search for casino on Netflix, its address bar reads"

And be replaced by the following text:

"If you want to add a search for a particular website to Flow, first enter a dummy text string in the search bar of that website, and launch the search. For example, if you search for casino on Netflix, its address bar reads"

221128 FL 1 10 0 (11 28)_Configuarción de las fuentes de búsqueda_Redundante

**Pull request merged**